src/SkiddleRequest.php

```diff
@@ -14,27 +14,71 @@ class SkiddleRequest
 {
 
     /**
-     * @var The version of the API we are currently using
+     * @var string The version of the API we are currently using
      */
     const API_VER = "v1";
 
     /**
-     * @var The URL we want to make calls to
+     * @var string The URL we want to make calls to
      */
     const API_URL = 'https://www.skiddle.com/api/';
 
     /**
-     * @var Whether we are in dev mode or not
+     * @var string The URL we want to make calls to (new api)
+     */
+    const API_V3_URL = 'https://api.skiddle.com/v3';
+
+    /**
+     * @var bool Whether we are in dev mode or not
      */
     public $dev_mode;
 
+    /**
+     * @var int Switch between different api versions
+     */
+    public $apiVersion = 1;
+
     /**
      * Basic constructor
      */
     public function __construct()
     {
     }
 
+    public function setApiVersion(int $apiVersion): void
+    {
+        $this->apiVersion = $apiVersion;
+    }
+
+    public function getApiVersion(): int
+    {
+        return $this->apiVersion;
+    }
+
+    /**
+     * v3 API - forces camelcasing of request & response data
+     * @param array  $requestData request keys to replace
+     * @return array The formatted results
+     */
+    public function transformRequestKeys($requestData)
```

Comment:
NIT: Could we add type-hinting here, please?

```suggestion
    public function transformRequestKeys(array $requestData): array
```